synapse/tests/test_model_inet.py

```diff
@@ -1365,6 +1377,15 @@ async def test_url_file(self):
             }})
             self.eq(t.norm(url), expected)
 
+            url = 'file:///c:/Users/BarUser/stuff/moar/stuff.txt'
```

Comment:
Kept distinct from the above case of `file://localhost/c:/Users/BarUser/stuff/moar/stuff.txt` because you can do silly things with DNS to override a name like `localhost` so that the host won't actually point to a local file.